.gitignore

```diff
@@ -21,6 +33,28 @@ DerivedData
 *.xcuserstate
 project.xcworkspace
 
+# Android Studio/IJ
```

Comment:
Not sure if this is all necessary, but should you put everything from android-teads-sdk git ignore here as well?
https://github.com/ebuzzing/service-inapp-sdk-android/blob/master/.gitignore